test/per_file_coverage.sh

```diff
@@ -18,6 +18,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/protobuf:94.6"
 "source/common/signal:84.5" # Death tests don't report LCOV
 "source/common/singleton:95.1"
+"source/common/stream_info:96.4"
```

Comment:
Why is this needed? Is this because you brought down the line-count making the coverage % worse? Any chance we could add coverage for missing lines?

Reply:
Yes, this is unfortunate.

> https://storage.googleapis.com/envoy-pr/88caf8e/coverage/source/common/stream_info/index.html
> 
> The coverage is lower because I rewrite the verbose code

That could be done in separate PRs. It's likely that code from elsewhere will be touched

Reply:
@alyssawilk for judgement on this point. @lambdai would you need to change prod code or just add unit tests.

Reply:
This is a legit reason to add an exception but given @lambdai is a frequent contributor I'd ask we just add unit tests here, even if they're for unrelated code.

testing upstreamSslConnection and serializeAsProto should be super easy and also get us back over the bar.

Reply:
Thank you! I am adding now

Reply:
Awesome!  Thanks so much for being willing to :-)

Reply:
The coverage is over the required percentage now. I have reverted the override coverage value. Can you take another look?